cypress/e2e/01-referral-service/PageSpecs/cannotConnectFamilyPage.spec.js

```diff
@@ -1,6 +1,6 @@
 describe('Cannot connect family to service page - FHG-2853', ()=> {
 	beforeEach(()=> {
-		cy.visit('https://dev.connect-families-to-support.education.gov.uk');
+		cy.visit('https://test.connect-families-to-support.education.gov.uk');
```

Comment:
Can we use cy.visit('/') instead , so the base Url gets picked from the config set up. 
When running UI mode we can use custom commands yarn run open:referral-dev / yarn run open:referral-test

Reply:
Changed base url in config.js. Also, updated spec files to use base url or pick the url from custom commands